browser/app/js/components/Browse.js

```diff
@@ -480,338 +486,309 @@ export default class Browse extends React.Component {
-                               id="file-input"></input>
-                             <label htmlFor="file-input"> <i className="fa fa-cloud-upload"></i> </label>
-                           </a>
-                         </OverlayTrigger>
-                         <OverlayTrigger placement="left" overlay={ makeBucketTooltip }>
-                           <a href="#" className="feba-btn feba-bucket" onClick={ this.showMakeBucketModal.bind(this) }><i className="fa fa-hdd-o"></i></a>
-                         </OverlayTrigger>
-                       </Dropdown.Menu>
-                     </Dropdown>
+    if (web.LoggedIn ()) {
+      createButton =  <Dropdown dropup className="create-new" id="dropdown-create-new">
+        <Dropdown.Toggle noCaret className="create-new__toggle"><i className="zmdi zmdi-plus"></i></Dropdown.Toggle>
+        <Dropdown.Menu>
+          <OverlayTrigger placement="top" overlay={ tooltips.uploadFile }>
+            <a href="#" className="create-new__btn create-new__btn--upload">
+              <input type="file" onChange={ this.uploadFile.bind (this) } id="object-upload-input" />
+              <label htmlFor="object-upload-input"> </label>
+            </a>
+          </OverlayTrigger>
+
+          <OverlayTrigger placement="top" overlay={ tooltips.createBucket }>
+            <a href="#" className="create-new__btn create-new__btn--bucket" onClick={ this.showMakeBucketModal.bind (this) }></a>
+          </OverlayTrigger>
+
+          <OverlayTrigger placement="top" overlay={ tooltips.uploadFolder }>
+            <a href="#" className="create-new__btn create-new__btn--folder"></a>
+          </OverlayTrigger>
+        </Dropdown.Menu>
+      </Dropdown>
 
     } else {
       if (prefixWritable)
-        createButton = <Dropdown dropup className="feb-actions" id="fe-action-toggle">
-                         <Dropdown.Toggle noCaret className="feba-toggle">
-                           <span><i className="fa fa-plus"></i></span>
-                         </Dropdown.Toggle>
-                         <Dropdown.Menu>
-                           <OverlayTrigger placement="left" overlay={ uploadTooltip }>
-                             <a href="#" className="feba-btn feba-upload">
-                               <input type="file"
-                                 onChange={ this.uploadFile.bind(this) }
-                                 style={ { display: 'none' } }
-                                 id="file-input"></input>
-                               <label htmlFor="file-input"> <i className="fa fa-cloud-upload"></i> </label>
-                             </a>
-                           </OverlayTrigger>
-                         </Dropdown.Menu>
-                       </Dropdown>
+        createButton =  <Dropdown dropup className="create-new" id="dropdown-create-new">
+          <Dropdown.Toggle noCaret className="create-new__toggle"><i className="zmdi zmdi-times"></i></Dropdown.Toggle>
```

Comment:
![add](https://user-images.githubusercontent.com/13393018/28409030-03712ed2-6d57-11e7-8def-566a0a87dcdb.png)

It looks fine on my Chrome on MacOS. What is your OS/Browser?